
From: Hariprasad Nellitheertha <hari@in.ibm.com>

There was a buggy (and unnecessary) reserve_bootmem call in the kdump call
which was causing hangs during early on some SMP machines.  The attached
patch removes that.

Signed-off-by: Hariprasad Nellitheertha <hari@in.ibm.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-i386/crash_dump.h |    1 -
 1 files changed, 1 deletion(-)

diff -puN include/asm-i386/crash_dump.h~crashdump-memory-preserving-reboot-using-kexec-fix include/asm-i386/crash_dump.h
--- 25/include/asm-i386/crash_dump.h~crashdump-memory-preserving-reboot-using-kexec-fix	Thu Nov 18 14:59:25 2004
+++ 25-akpm/include/asm-i386/crash_dump.h	Thu Nov 18 14:59:25 2004
@@ -29,7 +29,6 @@ static inline void set_saved_max_pfn(voi
 static inline void crash_reserve_bootmem(void)
 {
 	if (!dump_enabled) {
-		reserve_bootmem(0, CRASH_RELOCATE_SIZE);
 		reserve_bootmem(CRASH_BACKUP_BASE,
 			CRASH_BACKUP_SIZE + CRASH_RELOCATE_SIZE);
 	}
_
